src/style.css

```diff
@@ -0,0 +1,96 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap');
+
+body {
+  background-color: #f6f6f6;
+  font-family: 'Open Sans', sans-serif;
+}
+
+#list-container {
```

Comment:
- [ ] It would be better if we center the app as we have in the design. Doing this will help us achieve the design specification